flexmeasures/api/v3_0/tests/test_sensors_api.py

```diff
@@ -160,3 +160,28 @@ def test_patch_sensor_from_unrelated_account(client, setup_api_test_data):
 
     assert response.status_code == 403
     assert response.json["status"] == "INVALID_SENDER"
+
+
+def test_delete_a_sensor(client, setup_api_test_data):
+
+    existing_sensor_id = setup_api_test_data["some temperature sensor"].id
+    headers = make_headers_for("test_admin_user@seita.nl", client)
+    sensor_data = TimedBelief.query.filter(
+        TimedBelief.sensor_id == existing_sensor_id
+    ).all()
+    sensor_count = len(Sensor.query.all())
+
+    assert sensor_data[0].event_value == 815.0
```

Comment:
This assertion is specific (might break this test if the temperature data fixture slightly changes) and not needed to evaluate if deletion is working well.

Maybe we should simply assert that sensor_data is not empty?

Reply:
Thanks for the review. I've changed it to check for a float.